rest_framework/templatetags/rest_framework.py

```diff
@@ -154,7 +154,9 @@ def urlize_quoted_links(text, trim_url_limit=None, nofollow=True, autoescape=Tru
 
     If autoescape is True, the link text and URLs will get autoescaped.
     """
-    trim_url = lambda x, limit=trim_url_limit: limit is not None and (len(x) > limit and ('%s...' % x[:max(0, limit - 3)])) or x
+    def trim_url(x, limit=trim_url_limit):
+        return limit is not None and (len(x) > limit and ('%s...' % x[:max(0, limit - 3)])) or x
```

Comment:
FWIW, I'd feel better if these `conditional and y or x` lines were replaced with explicit Python conditional assignment statements.

``` python
return ('%s...' % x[:max(0, limit - 3)]) if limit and len(x) > limit else x
```